bothan-api/server/src/api/crypto.rs

```diff
@@ -8,30 +8,34 @@ use bothan_core::manager::crypto_asset_info::error::{PushMonitoringRecordError,
 use bothan_core::manager::CryptoAssetInfoManager;
 
 use crate::api::utils::parse_price_state;
-use crate::proto::price::price_service_server::PriceService;
-use crate::proto::price::{GetPricesRequest, GetPricesResponse, Price};
-use crate::proto::signal::signal_service_server::SignalService;
-use crate::proto::signal::{GetInfoResponse, PushMonitoringRecordsRequest, UpdateRegistryRequest};
+use crate::proto::bothan::v1::{
+    BothanService, GetInfoRequest, GetInfoResponse, GetPricesRequest, GetPricesResponse, Price,
+    PushMonitoringRecordsRequest, PushMonitoringRecordsResponse, UpdateRegistryRequest,
+    UpdateRegistryResponse,
+};
 
 pub const PRECISION: u32 = 9;
 
 /// The `CryptoQueryServer` struct represents a server for querying cryptocurrency prices.
 pub struct CryptoQueryServer {
-    manager: Arc<CryptoAssetInfoManager<'static>>,
+    crypto_manager: Arc<CryptoAssetInfoManager<'static>>,
```

Comment:
don't change file name 'crypto' to something else ror?

Reply:
why keep crypto na?

Reply:
will rename server and file name but will keep internal as crypto_manager